server.go

```diff
@@ -98,12 +98,25 @@ func runMain() error {
 		}
 	}
 
+	mainControlParam := controllers.MainControllerParams{
+		AdminIPs:        cfg.AdminIPs,
+		AdminUserIDs:    cfg.AdminUserIDs,
+		APISecret:       cfg.APISecret,
+		BaseURL:         cfg.BaseURL,
+		ClosePool:       cfg.ClosePool,
+		ClosePoolMsg:    cfg.ClosePoolMsg,
+		PoolEmail:       cfg.PoolEmail,
+		PoolFees:        cfg.PoolFees,
+		PoolLink:        cfg.PoolLink,
+		RealIPHeader:    cfg.RealIPHeader,
+		MaxVotedTickets: cfg.MaxVotedTickets,
+		Description:     cfg.Description,
+		Designation:     cfg.Designation,
+	}
+
 	controller, err := controllers.NewMainController(activeNetParams.Params,
-		cfg.AdminIPs, cfg.AdminUserIDs, cfg.APISecret, APIVersionsSupported,
-		cfg.BaseURL, cfg.ClosePool, cfg.ClosePoolMsg, coldWalletFeeKey,
-		stakepooldConnMan, cfg.PoolFees, cfg.PoolEmail, cfg.PoolLink,
-		sender, cfg.RealIPHeader, votingWalletVoteKey,
-		cfg.MaxVotedTickets, cfg.Description, cfg.Designation)
+		mainControlParam, APIVersionsSupported, coldWalletFeeKey,
```

Comment:
why are `APIVersionsSupported`, `coldWalletFeeKey` (and others) being left out of the new config struct?

Reply:
They have been included in the new commit.